code/modules/mob/living/carbon/alien/humanoid/queen.dm

```diff
@@ -61,10 +104,12 @@
 
 /mob/living/carbon/alien/humanoid/royal/queen/death()//yogs start: dead queen doesnt stop shuttle
```

Comment:
move your qdel to the proper place and delete this death and destroy, you do it in the [code/modules/mob/living/carbon/alien/humanoid/death.dm](https://github.com/yogstation13/Yogstation/pull/14143/files#diff-3d6442b71f8c6373464f479ceb75f0e90a1e82b8daac7122a5b0df49545f9d36)

Reply:
Don't understand